components/tabs/skin.css

```diff
@@ -19,12 +19,26 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Tabs-selectionIndicator {
-  background-color: var(--spectrum-tabs-textonly-tabitem-selection-indicator-background-color-selected);
+  --spectrum-tabs-list-background-direction: right;
+
+  background: linear-gradient(
+    to var(--spectrum-tabs-list-background-direction),
+    var(--spectrum-tabs-textonly-tabitem-selection-indicator-background-color-selected) 0,
+    var(--spectrum-tabs-textonly-tabitem-selection-indicator-background-color-selected)
+    var(--spectrum-tabs-quiet-textonly-divider-size),
+    transparent var(--spectrum-tabs-quiet-textonly-divider-size)
```

Comment:
What token should I use in place of `--spectrum-tabs-quiet-textonly-divider-size`?

The widths are currently set as an inline styles, so I'm not sure the purpose of this token, which is only 2px.

<img width="1765" alt="Screen Shot 2022-09-12 at 9 02 08 AM" src="https://user-images.githubusercontent.com/16216104/189674063-c5d18cfb-23dd-4d03-87cf-c49f08a305b0.png">

<img width="1229" alt="Screen Shot 2022-09-12 at 9 01 52 AM" src="https://user-images.githubusercontent.com/16216104/189674067-bc93ffaf-a33a-4b27-a8cc-4fd8fef6d058.png">

Reply:
Thanks Patrick, I misunderstood how the width is was supposed to work and you helped clarify that.

I updated the PR and it looks correct now. Can I reproduce the issue in the the PR link to make sure this works as expected?